lib/components/parallax_component.dart

```diff
@@ -201,12 +203,8 @@ class ParallaxComponent extends PositionComponent {
     if (!loaded()) {
       return;
     }
-
     super.render(canvas);
-
-    canvas.save();
     _layers.forEach((layer) => layer.render(canvas));
-    canvas.restore();
```

Comment:
is this safe to remove?

Reply:
Maybe not? I removed it as part of finding where the bug was and forgot to add it again.

Reply:
Added them back